### PR TITLE
Add automation to aggregate labs into the gitbook

### PR DESCRIPTION
**Pull request by mlangbehn**
Signed-off-by: Matt Langbehn <matthew.langbehn@gmail.com>

### CODE REVIEW

**Review by jjasghar — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
-matrix:
-  include:
-    - language: node_js
-      node_js: 10
-      script:
-        - npm install markdownlint-cli
-        - markdownlint --config .markdownlint.json --ignore workshop/SUMMARY.md --ignore workshop/README.md workshop
+---
```
Comment (stevemar) [inline .travis.yml:1]: oops, thanks for cleaning this up

File: aggregate-labs.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	git clone "$repo" tempClones/"$workshopName"
+	cp -a tempClones/"$workshopName"/workshop workshop/generatedContent/"$workshopName"
+	echo "* [generatedContent/$workshopName](generatedContent/$workshopName/README.md)" >> workshop/generatedContentLinks.md
+	for lab in workshop/generatedContent/"$workshopName"/*/*; do
+        [[ -d "$lab" ]] || break
+		labName=$(basename "$lab")
+		echo "    * [generatedContent/$workshopName/$labName](generatedContent/$workshopName/$labName/README.md)" >> workshop/generatedContentLinks.md
+		echo "Lab -> $(basename "$lab")"
+	done
+	echo "This content is generated! Do not edit directly! Please run aggregateStuff.sh to repopulate with latest content from agenda.txt!" > workshop/generatedContent/README.md
```
Comment (stevemar) [inline aggregate-labs.sh:21]: old file name here... s/aggregateStuff.sh/aggregate-labs.sh

File: .markdownlintignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+workshop/SUMMARY.md
```
Comment (stevemar) [inline .markdownlintignore:1]: ah, this is new! cool! https://github.com/DavidAnson/markdownlint/issues/245

**Comment by stevemar** [comment]
@mlangbehn @jjasghar beat me to the merge, just caught one old reference, looks good overall though.

**Pull request merged**